lte/gateway/python/magma/mobilityd/main.py

```diff
@@ -14,6 +14,7 @@
 import logging
 from typing import Any, Optional
 
+import sdnotify
 from lte.protos.mconfig import mconfigs_pb2
 from lte.protos.mconfig.mconfigs_pb2 import MobilityD
 from lte.protos.subscriberdb_pb2_grpc import SubscriberDBStub
```

Comment:
lets add the notification to subscriberDB since mobilityD dependency.

Reply:
There does not seem to be a systemd dependency in orc8r/tools/ansible/roles/gateway_services/files/magma.service for mobilityd and directoryd. How is that handled?